fury/actor.py

```diff
@@ -3954,3 +3956,152 @@ def uncertainty_cone(
     angles = main_dir_uncertainty(evals, evecs, signal, sigma, b_matrix)
 
     return double_cone(centers, evecs, angles, colors, scales, opacity)
+
+
+class TexturedCube:
```

Comment:
We prefer to avoid class and be more functional. What is the advantage of using a class here?